adjust_restart_for_new_land_cover/adjust_restart_for_new_land_cover.py

```diff
@@ -220,45 +215,49 @@ def remap_vegetation(
     MinPointsFound = MappingConf['minimum_points']
     
     # Set up the Dataset we're going to write to
-    OutDataset = setup_output_dataset(OutputVegetation, PerCellVariables +
-                                      PerTileVariables)
+    OutDataset = setup_output_dataset(OutputVegetation, InputDataset,
+                                      PerCellVariables + PerTileVariables)
```

Comment:
```suggestion
    OutDataset = setup_output_dataset(OutputVegetation)
```
These arguments might be able to be removed

Reply:
Yep, removed.